packages/ws-server/src/webSocketServer.ts

```diff
@@ -43,6 +42,8 @@ const register = new Registry();
 const relay: Relay = new RelayImpl(logger, register);
 let connectedClients = 0;
 const MAX_CONNECTIONS = parseInt(process.env.CONNECTION_LIMIT || '10');
+// 60 minutes in ms by default
+const MAX_CONNECTION_TTL = parseInt(process.env.WS_MAX_CONNECTION_TTL || '3600000'); // 60 minutes in ms by default
```

Comment:
Can you confirm this with the design doc.
I feel like we noted this somewhere.

Reply:
Yes indeed, it is on the eth_subscribe design doc and is set to 5 minutes by default.
✅  done.